File: e2e/java/src/main/java/io/trzcinski/test/pet/dto/Pet.java
package io.trzcinski.test.pet.dto;

import io.trzcinski.test.commons.dto.Category;
import io.trzcinski.test.commons.dto.Tag;
import lombok.Data;
import org.springframework.lang.Nullable;
import java.util.List;

@Data
public class Pet{
    @Nullable
    private Integer id;
    @Nullable
    private Category category;
    private String name;
    private List<String> photoUrls;
    @Nullable
    private List<Tag> tags;
    @Nullable
    private String status;
}

File: test-samples/kotlin-retrofit/src/main/kotlin/io/trzcinski/test/pet/dto/Pet.kt
package out.io.trzcinski.test.pet.dto

import out.io.trzcinski.test.commons.dto.Category
import out.io.trzcinski.test.commons.dto.Tag

data class Pet(
    val id: Int?,
    val category: Category?,
    val name: String,
    val photoUrls: List<String>,
    val tags: List<Tag>?,
    val status: String?,
)